.github/workflows/build.yaml

```
@@ -0,0 +1,49 @@
+name: Build and verify
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [master]
```

```suggestion
    branches: [$default-branch]
```

We could go for this?